packages/lesswrong/server/callbacks/commentCallbacks.ts

```diff
@@ -95,23 +95,22 @@ getCollectionHooks("Comments").newSync.add(async function CommentsNewOperations
   if (comment.postId) {
     const lastCommentedAt = new Date()
 
-    if (!comment.debateResponse) {
-      await Posts.rawUpdateOne(comment.postId, {
-        $set: {lastCommentedAt},
-      })
-    }
-
     // we're updating the comment author's lastVisitedAt time for the post as well,
     // so that their comment doesn't cause the post to look like it has unread comments
-    await ReadStatuses.rawUpdateOne({
-      postId: comment.postId,
-      userId: comment.userId,
-      tagId: null,
-    }, {
-      $set: {
-        lastUpdated: lastCommentedAt
-      }
-    })
+    await Promise.all([
+      !comment.debateResponse ? Posts.rawUpdateOne(comment.postId, {
```

Comment:
I'd probably assign the promise to its own variable, and not use `null`:

```
const updateLastCommentedAtPromise = comment.debateResponse ? Promise.resolve() : Posts.rawUpdateOne(...);

await Promise.all([updateLastCommentedAtPromise, ReadStatuses.rawUpdateOne(...)]);
```

Maybe extract the other one too just for readability.